### ISSUE TITLE
Multiline description formatting beyond it's scope

### ISSUE THREAD

**Issue by NickHargy** [swagger-api/swagger-editor#2828]
It's been a while since I looked at this documentation but since then the multi-line descriptions in Swagger Editor seem to not know their place.

It's apparent to me that the .api functionality remains, warnings are thrown as and when they're supposed to be, but the text is all green, all the way up until the parent path is exited.

![grafik](https://user-images.githubusercontent.com/37898881/141987759-fc183a42-bdda-482a-82a5-5e50fc8ce28f.png)

...2000 lines later....

![grafik](https://user-images.githubusercontent.com/37898881/141987537-49faa9b2-5b80-4d51-94e1-fd2c4b14f405.png)

Inconvenient. 

I have tried reimporting the file and using different formatting structures; The only work around I have found so far is to use quotation marks, but this is less than ideal in certain situations.

**Comment by Mike4Online** [comment]
Re: loss of syntax highlighting after specifying a multi-line description in YAML

I've confirmed this is occurring in Swagger Editor 4.0.1.  This issue can be replicated easily on [https://editor.swagger.io/](https://editor.swagger.io/) using the sample YAML file, in particular when setting a multi-line description for a **path** item, when specifying a vertical bar (pipe) as the first line of the description, i.e. the block style indicator described [here](https://yaml-multiline.info/).  For such items, the work-around of using quotes does not seem to help restore proper syntax highlighting.

**Comment by davidstosik** [comment]
Yes, I have the problem too. As mentioned, it can be reproduced on editor.swagger.io.

<details><summary>Here's a minimal YAML file to reproduce.</summary>

```yaml
swagger: "2.0"
info:
  description: |
    This is a sample demonstrating the multi-line syntax highlighting problem
    One more line.
    
    Then another one after an empty one.

  version: "1.0.0"
  title: "Swagger multiline syntax highlight"
paths: {}
```

</details>

Check the `version:` and `title:` keys in the screenshot below:

<img width="698" alt="Untitled 3" src="https://user-images.githubusercontent.com/816901/145745020-db6abcec-159e-4c0e-8f16-1307fc9bafc4.png">

Compared to what they should look like:

<img width="806" alt="Untitled 4" src="https://user-images.githubusercontent.com/816901/145745081-fde4c284-7849-45f7-8a63-93b7e2360414.png">

(Also shows a workaround.)

---

I thought this bug might be coming from Brace, but I wasn't able to reproduce with Brace 0.11.1. 🤔

**Comment by Skeeve** [comment]
(Ugly) workaround: Place a lonely "#" as first character in a new line after the string.

**Comment by MichakrawSB** [comment]
We’ve released Swagger Editor v5! 🎉 Check out the details here: [Inside the New Swagger Editor](https://swagger.io/blog/inside-the-new-swagger-editor/).
We’re closing old issues related to previous versions. If you think any of them are still relevant, please open a new issue – this helps us prioritize what matters most. Thanks for your activity! 🚀
